src/bin/wasm-tools/component.rs

```diff
@@ -308,6 +309,10 @@ pub struct EmbedOpts {
     /// Print the output in the WebAssembly text format instead of binary.
     #[clap(long, short = 't')]
     wat: bool,
+
+    /// Print the wasm custom section only.
+    #[clap(long, short = 's')]
+    only_custom: bool,
```

Comment:
I'm a bit wary to too eagerly reserve short flags so mind leaving this as just `long`?

Reply:
Also mind adding some `conflicts_with` annotations to indicate that it doesn't work with flags such as `--wat`, `--dummy`, etc?